packages/react/src/components/tabs/tabs.tsx

```diff
@@ -142,6 +147,7 @@ export const Tabs: VoidFunctionComponent<Props> = ({
     forceRenderTabPanels,
     tabs,
     defaultSelectedId,
+    providedAddButtonProps,
```

Comment:
PY voulais dire de renommer a providedAddButtonProps a l'interne seulement. Le props garde le meme nom.

```suggestion
    addButtonProps: providedAddButtonProps,
```